Automattic/sensei
# Update Learning Mode texts

Fixes #4689

### Changes proposed in this Pull Request

* It updates the Learning Mode texts.

### Testing instructions


* Check the course editor sidebar, and make sure the texts are correct based on [the issue](https://github.com/Automattic/sensei/issues/4689).
* Go to WP Admin > Sensei LMS > Settings > Courses.
* Make sure the copy matches the course settings copy idea.


### Screenshot / Video

<img width="308" alt="Screen Shot 2022-01-26 at 17 21 15" src="https://user-images.githubusercontent.com/876340/151241153-843cdd7b-c8b7-4203-a04a-4cb2e7c38963.png">

<img width="1111" alt="Screen Shot 2022-01-26 at 17 21 23" src="https://user-images.githubusercontent.com/876340/151241170-a95d519a-9d89-4a14-b571-c41cc9f88b8b.png">
